pyproject.toml

```diff
@@ -20,16 +20,23 @@ dependencies = [ "pydantic>=2" ]
 
 [project.optional-dependencies]
 django = ["django>=4.2"]
+flask = ["flask>=2"]
+sqlalchemy = ["sqlalchemy>=2.0"]
```

Comment:
This would make the install of sqlalchemy optional. If wanted a user could do `pip install tidewave[flask,sqlalchemy]`. Which seems good, there's good use cases for Flask without sqlalchemy.